src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
I see 2 options:

1) Directly reference the older Runtime package in tool-runtime project.json which will result in a downgrade that we can ignore.
2) Update either manually or via PR in CoreCLR repo to point the ILAsm package at the older version of runtime package (or potentially even remove the dependency).